src/actions/index.js

```diff
@@ -217,3 +222,16 @@ export const generateGIF = (images, opts) => (dispatch, getState) => {
     }
   });
 };
+
+export const getBurstSliders = () => dispatch => {
+  const sliders = getSliderExpressions();
+  if (!sliders.length) {
+    dispatch(flashError(noSlidersFound()));
+  }
+  dispatch(updateBurstSliders(sliders));
+};
+
+const updateBurstSliders = sliders => ({
```

Comment:
In the rest of this file, the actions appear to be at the top. Is there a particular reason you've placed this one apart from the rest?

Reply:
good catch! moved from thunk section to action section